eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -1,13 +1 @@
-steps:
-  - task: UseDotNet@2
-    displayName: 'Use .NET Core SDK'
-    condition: ne(variables['Agent.OS'], 'Windows_NT') # Windows supports MultiLevelLookup and doesn't need explicit framework installation
-    inputs:
-      useGlobalJson: true
-      performMultiLevelLookup: true
-  - task: UseDotNet@2
-    condition: ne(variables['Agent.OS'], 'Windows_NT') # Windows supports MultiLevelLookup and doesn't need explicit framework installation
-    displayName: 'Use .NET Core 2.1 runtime'
-    inputs:
-      packageType: runtime
-      version: "2.1.x"
+steps: []
```

Comment:
FYI: @mitchdenny 

@pakrym we should at least add a comment here. We might even want to consider adding a verification step.

Reply:
Lately, the installation step was fast and non-flaky. Should we keep it instead of the verification?

Reply:
The way I would do this is comment the tasks out with a brief comment at the top explaining that they can be commented back in when a new version of .NET is released and the hosted agents don't yet have the bits. I think this is something we'll oscillate on a lot.

Reply:
Added comments/kept commented out steps.